libs/output-mapping/src/Storage/StoragePreparer.php

```diff
@@ -24,6 +25,7 @@ public function __construct(readonly private ClientWrapper $clientWrapper, reado
     public function prepareStorageBucketAndTable(
         MappingFromProcessedConfiguration $processedSource,
         SystemMetadata $systemMetadata,
+        bool $hasNewNativeTypeFeature = false,
```

Comment:
Bych řekl, že by to mohlo být přímo v constructoru? 🤔 u `TableStructureValidator` to taky tak máme.
Nenapadá mě použití kdy by se to volalo dvakrát a pokaždé s jiným flagem.

Reply:
může být